packages/neuron-wallet/src/database/chain/migrations/1690361215400-ResetSyncProgressPrimaryKey.ts

```diff
@@ -1,15 +1,15 @@
 import {MigrationInterface, QueryRunner} from "typeorm";
-import SyncProgress from "../entities/sync-progress";
 
+const chunk = 100
 export class ResetSyncProgressPrimaryKey1690361215400 implements MigrationInterface {
     name = 'ResetSyncProgressPrimaryKey1690361215400'
 
     public async up(queryRunner: QueryRunner): Promise<void> {
-        const syncProgresses = await queryRunner.manager.find(SyncProgress)
+        const syncProgresses = await queryRunner.manager.query('select * from sync_progress')
         await queryRunner.query(`DROP TABLE "sync_progress"`)
         await queryRunner.query(`CREATE TABLE "sync_progress" ("hash" varchar NOT NULL, "args" varchar NOT NULL, "codeHash" varchar NOT NULL, "hashType" varchar NOT NULL, "scriptType" varchar NOT NULL, "walletId" varchar NOT NULL, "blockStartNumber" integer NOT NULL, "blockEndNumber" integer, "cursor" varchar, "delete" boolean, "addressType" integer, PRIMARY KEY ("hash", "walletId"))`)
-        for (let index = 0; index < syncProgresses.length; index += 500) {
-          await queryRunner.manager.save(syncProgresses.slice(index, index + 500))
+        for (let index = 0; index < syncProgresses.length; index += chunk) {
```

Comment:
Is there any reason for modifying a published migration? The migration should not execute after 1690361215400(2023-07-26T08:46:55.400Z)

Reply:
Ignore this, I found the answer here https://github.com/nervosnetwork/neuron/pull/2992/files#r1437411523